drivers/can/Kconfig

```diff
@@ -60,14 +60,15 @@ config CANFD_MAX_DLC
 	  optimize memory consumption.
 endif # CAN_FD_MODE
 
-config CAN_INIT_PRIORITY
-	int "CAN driver init priority"
-	default 80
+config CAN_SOCKETCAN_NETDEVICE_INIT_PRIORITY
+	int "socketCAN net device init priority"
+	default 81
+	depends on NET_SOCKETS_CAN
 	help
-	  CAN device driver initialization priority.
+	  socketCAN net device initialization priority.
 	  Do not mess with it unless you know what you are doing.
 	  Note that the priority needs to be lower than the net stack
-	  so that it can start before the networking sub-system.
+	  and CAN driver so that it can start before the networking sub-system.
```

Comment:
It needs to initialise _after_ the CAN driver, but _before_ the network subsystem, right?

Reply:
Yep, think so